_build.ham

```diff
@@ -0,0 +1,16 @@
+SubDir TOP ;
+
+local BIN_DIR = [ FDirName $(TOP) bin ] ;
+# ECHO "... BIN_DIR:" $(BIN_DIR) ;
+
+# Recursively get all .sh files inside of bin/ directory
+local SH_FILES = [ FGristFiles [ ListFilesR $(BIN_DIR) : *.sh ] ] ;
```

Comment:
ListFilesR always makes me nervous, but its there for a reason I guess ;) Its probably fine here.